samples/vanilla/README.md

```diff
@@ -0,0 +1,17 @@
+# Vanilla + Refinitiv-ui
+
+This project was generated without the use of framework.
+
+## Getting started
+
+Install dependencies and deduplicate packages.
+
+```sh
+  npm install && npm dedupe
```

Comment:
I don't think we need to dedupe packages in V5.

Reply:
Agree. I will remove it.